src/main/lrsql/system/database.clj

```diff
@@ -97,7 +97,7 @@
       (if-not ?conn-pool
         (let [conn-pool (make-conn-pool backend config)]
           (log/infof "Starting new connection for %s database..." db-type)
-          (log/debugf "Config: %s" config)
+          (log/debugf "Config: %s" (cu/redact-config-vars config))
```

Comment:
After reading the `c.t.l` source, the walk is fine. If this only happens in debug, the expanded macro contains a conditional on level.